packages/now-static-build/src/frameworks.ts

```diff
@@ -82,7 +82,7 @@ const frameworkList: Framework[] = [
     name: 'Docusaurus',
     slug: 'docusaurus',
     dependency: '@docusaurus/core',
-    buildCommand: 'docusaurus-build',
+    buildCommand: 'docusaurus build',
```

Comment:
We need to differentiate docusaurus 1 and docusaurus 2.

Reply:
We only support Docusaurus 2 for detection here